VideoWeb/VideoWeb/ClientApp/src/app/app-routing.module.ts

```diff
@@ -29,6 +29,6 @@ export const routes: Routes = [
 
 @NgModule({
     exports: [RouterModule],
-    imports: [RouterModule.forRoot(routes, { enableTracing: !environment.production })]
+    imports: [RouterModule.forRoot(routes, { enableTracing: environment.production })]
```

Comment:
why the switch to production?

Reply:
oh need to revert this! Thanks. it was too noisy during my testing